src/odemis/acq/align/autofocus.py

```diff
@@ -57,6 +57,27 @@ def _convertRBGToGrayscale(image):
     return gray
 
 
+def AssessFocus(levels):
+    """
+    Given a list of focus levels, it decides if there is any significant value
+    or it only contains noise.
```

Comment:
please define "level" parameter. list of floats? with the biggest meaning better focus?